CODEOWNERS

```diff
@@ -27,6 +27,10 @@
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/janitors
+/.github/ @cilium/contributing
+/.github/workflows/ @cilium/ci
+/.github/workflows/coccicheck.yaml @cilium/ci @cilium/bpf
+/.github/workflows/docs.yaml @cilium/ci @cilium/docs
```

Comment:
Not sure I should be adding @cilium/ci for these last two directories. That's consistent with other test directories in `CODEOWNERS`, but @cilium/ci gets pinged for so many PRs now that I'm not sure it makes sense anymore.

Reply:
I think it should be under oversight of @cilium/ci, especially in the interest of consistency and reliability.